Readme.md

```diff
@@ -1,25 +1,111 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128534875/19.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1760)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to implement select/unselect for all rows in a group row
+# Grid View for ASP.NET Web Forms- How to select/deselect all rows in a group when the control is grouped by a column
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e1760/)**
 <!-- run online end -->
+This example demonstrates how to allow users to add/remove all rows in a group to/from selection. Note that this approach works only when the [Grid View](https://docs.devexpress.com/AspNet/5823/components/grid-view?p=netframework)'s data is grouped by a single column.
```

Comment:
> Note that this **approach** works only when the [Grid View](https://docs.devexpress.com/AspNet/5823/components/grid-view?p=netframework)'s data is grouped by a single column.

Approach. Usage of this word is being discouraged of late. I prefer "technique" instead.

Is there a specific reason to use the following form?

> the Grid View's data

Instead of "Grid View data"

> is grouped by a **single** column.

Tautology. "A" already means "single", actually. Therefore, "a single" makes extra emphasis on the aspect of singularity. I understand that "is grouped by a column" sounds somewhat strange, and that's what caused you to add "single". I recommend replacing "a single" with "one" instead because it describes our scenario.

```suggestion
This example demonstrates how to allow users to add/remove all rows in a group to/from selection. Note that this technique works only when [Grid View](https://docs.devexpress.com/AspNet/5823/components/grid-view?p=netframework) data is grouped by one column.
```

If you have any questions or doubts on this change - please let me know.